src/braket/circuits/gates.py

```diff
@@ -938,7 +1026,13 @@ def ry(
 
 
 class Rz(AngledGate):
-    """Z-axis rotation gate.
+    r"""Z-axis rotation gate.
+
+    Unitary matrix:
+        .. math:: \mathtt{R_z}(\phi) = \begin{bmatrix}
```

Comment:
Do we want to normalize the rotation gates to have theta as their represented angles?

Reply:
@math411 Suggest to use either phi or theta throughout?
Of course this is a nit.

Reply:
Renamed theta->phi whenever possible to unify notations